deno.jsonc

```
@@ -10,7 +10,7 @@
     }
   },
   "tasks": {
-    "test": "deno fmt --check && deno lint && deno test src",
-    "coverage": "deno test --coverage=.coverage src && deno coverage --exclude=fixtures --exclude=test .coverage"
+    "test": "deno fmt --check && deno lint && deno test src --allow-read",
```

Small suggestion, the flags should go before the directory parameter, just to align with `deno test` CLI help/usage docs.

```suggestion
    "test": "deno fmt --check && deno lint && deno test --allow-read src",
```